app/views/admin/shared/tagging/_legacy_associations_box.html.erb

```diff
@@ -6,41 +6,39 @@
       <% end %>
     <% end %>
   </h2>
-  <div class="content content-bordered">
-    <% if edition.can_be_related_to_policies? && tagged_policy_names(edition.policy_content_ids).any? %>
-      <h4>Policies</h4>
-      <ul class="policies">
-        <% tagged_policy_names(edition.policy_content_ids).each do |name| %>
-          <li><%= name %></li>
-        <% end %>
-      </ul>
-    <% end %>
-    <% if edition.can_be_associated_with_topics? %>
-      <h4>Policy Areas</h4>
-      <% if edition.topics.any? %>
+  <% if edition.has_legacy_tags? %>
+    <div class="content content-bordered">
+      <% if edition.has_policies? %>
+        <h4>Policies</h4>
+        <ul class="policies">
+          <% tagged_policy_names(edition.policy_content_ids).each do |name| %>
+            <li><%= name %></li>
+          <% end %>
+        </ul>
+      <% end %>
+      <% if edition.has_policy_areas? %>
+        <h4>Policy Areas</h4>
         <ul class="policy-areas">
           <% edition.topics.each do |name| %>
             <li><%= name %></li>
           <% end %>
         </ul>
 
-      <% else %>
-        <p><%= no_policy_areas_message %> </p>
       <% end %>
-    <% end %>
-    <% unless specialist_sector_name(edition.primary_specialist_sector_tag).blank? %>
-      <h4>Primary Specialist Sector</h4>
-      <ul class="primary-specialist-sector">
-        <li><%= specialist_sector_name(edition.primary_specialist_sector_tag) %></li>
-      </ul>
-    <% end %>
-    <% unless specialist_sector_names(edition.secondary_specialist_sector_tags).empty? %>
-      <h4>Secondary Specialist Sectors</h4>
-      <ul class="secondary-specialist-sectors">
-        <% specialist_sector_names(edition.secondary_specialist_sector_tags).each do |name| %>
-          <li><%= name %></li>
-        <% end %>
-      </ul>
-    <% end %>
-  </div>
+      <% if edition.has_primary_sector? %>
+        <h4>Primary Specialist Sector</h4>
+        <ul class="primary-specialist-sector">
+          <li><%= specialist_sector_name(edition.primary_specialist_sector_tag) %></li>
+        </ul>
+      <% end %>
+      <% if edition.has_secondary_sectors? %>
+        <h4>Secondary Specialist Sectors</h4>
+        <ul class="secondary-specialist-sectors">
+          <% specialist_sector_names(edition.secondary_specialist_sector_tags).each do |name| %>
+            <li><%= name %></li>
+          <% end %>
+        </ul>
+      <% end %>
+    </div>
+  <% end %>
```

Comment:
It might be interesting to look at using the "No content" pattern here, as shown in the `govuk_admin_template` style guide [1]

1: https://github.com/alphagov/govuk_admin_template/blob/release-v6.6.0/app/views/govuk_admin_template/style_guide/index.html.erb#L544-L553

Reply:
I've just seen that you've done that here 19f2425c4548ca5f511cf278fded58510b7c7c95